noxfile.py

```diff
@@ -1,9 +1,33 @@
 import nox
 
+
+@nox.session(name="pip-compile", python=["3.11"])
+def pip_compile(session: nox.Session):
```

Comment:
nit: https://nox.thea.codes/en/stable/config.html#session-description says that a docstring would be displayed in the `nox --list` output which is super useful for session discovery. So I suggest writing something up PEP 257-style.

Reply:
Thanks! Great suggestion.